app/src/main/java/org/mozilla/focus/home/TopSiteAdapter.java

```diff
@@ -85,7 +85,7 @@ public int getItemCount() {
     }
 
     public void addSite(int index, @NonNull Site toAdd) {
-       this.sites.add(index, toAdd);
+        this.sites.add(index, toAdd);
```

Comment:
Personally I would move this to another commit so this would not pollute this commit, but it's just a matter of preference.